example/app/config.yml

```diff
@@ -60,6 +60,25 @@ default:
     stepRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/{{prefix}}-integration-steprole'
     instanceProfile: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:instance-profile/{{prefix}}-integration-ecs'
 
+  sns:
+    inregions3:
+      subscriptions:
+        lambda:
+          endpoint:
+            function: Fn::GetAtt
+            array:
+              - InRegionS3PolicyLambdaFunction
+              - Arn
+    AmazonIpSpaceChanged:
+      arn: arn:aws:sns:us-east-1:806199016981:AmazonIpSpaceChanged
```

Comment:
I would take out the account number here.

Reply:
This isn't our account number, it's completely public and grabbed it from here: https://aws.amazon.com/blogs/aws/subscribe-to-aws-public-ip-address-changes-via-amazon-sns/